### PR TITLE
Markdown conversion for `ru` - Replace - Manifest section :warning: Do not squash :warning: 

### PR DESCRIPTION
**Pull request by SphinxKnight**
Follow-up of: #8559
Conversion report: https://gist.github.com/SphinxKnight/e66a41050cc8314ab9fcfd527ef8201f#file-md-conversion-problems-report-2022-09-29t11_16_11-196z-md

### CODE REVIEW

**Comment by SphinxKnight** [comment]
Please :warning: Do not squash merge :warning:

File: files/ru/web/manifest/name/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    <tr>
+      <th scope="row">Mandatory</th>
+      <td>Yes</td>
+    </tr>
+    <tr>
+      <th scope="row">Example</th>
+      <td>
+        <pre class="brush: json no-line-numbers">
+"name": "Awesome application"</pre
+        >
```
Comment (sashasushko) [inline files/ru/web/manifest/name/index.md:27]:
```suggestion
"name": "Awesome application"</pre>
```

File: files/ru/web/manifest/related_applications/index.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      <td>
+        <pre class="brush: json no-line-numbers">
+"related_applications": [
+  {
+    "platform": "play",
+    "url": "https://play.google.com/store/apps/details?id=com.example.app1",
+    "id": "com.example.app1"
+  }
+]</pre
+        >
```
Comment (sashasushko) [inline files/ru/web/manifest/related_applications/index.md:33]:
```suggestion
]</pre>
```

File: files/ru/web/manifest/screenshots/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+      <td>
+        <pre class="brush: json no-line-numbers">
+"screenshots": [
+  {
+    "src": "screenshot.webp",
+    "sizes": "1280x720",
+    "type": "image/webp"
+  }
+]</pre
+        >
```
Comment (sashasushko) [inline files/ru/web/manifest/screenshots/index.md:33]:
```suggestion
]</pre>
```

File: files/ru/web/manifest/start_url/index.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    <tr>
+      <th scope="row">Mandatory</th>
+      <td>No</td>
+    </tr>
+    <tr>
+      <th scope="row">Example</th>
+      <td>
+        <pre class="brush: json no-line-numbers">
+"start_url": "https://example.com"</pre
+        >
```
Comment (sashasushko) [inline files/ru/web/manifest/start_url/index.md:27]:
```suggestion
"start_url": "https://example.com"</pre>
```

**Pull request merged**